apps/ejabberd/src/ejabberd_router.erl

```diff
@@ -57,19 +53,18 @@
 -include("ejabberd.hrl").
 -include("jlib.hrl").
 
+-record(state, {}).
+
 -type handler() :: 'undefined'
-                | {'apply_fun',fun((_,_,_) -> any())}
```

Comment:
Curiouser and curiouser. Why was `'apply_fun'` used instead of `apply_fun`? (not related to this PR)